test/blink/2-keyframes-with-offsets.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
-<script src="testharness/testharness.js"></script>
-<script src="testharness/testharnessreport.js"></script>
+<script src="../web-platform-tests/resources/testharness.js"></script>
```

Comment:
My concern is that it may be confusing to come into the code and see some scripts imported from "/resources" and some from "resources". I think you could add the third one as well if you specified '/resources/keyframes-test.js' as an entry in the karma config, but that may very well be undesirable. An alternative may be to rename one of them so the imports don't just differ in the presence of a single slash.